OTAcceleratorCore/OTMultiPartyCommunicator.m

```diff
@@ -133,6 +133,9 @@ + (instancetype)sharedInstance {
 @interface OTMultiPartyCommunicator() <OTSessionDelegate, OTSubscriberKitDelegate, OTPublisherDelegate, OTVideoViewProtocol>
 @property (nonatomic) BOOL isCallEnabled;
 @property (nonatomic) NSString *name;
+@property (nonatomic) NSUInteger internalConnectionCount;
```

Comment:
This line should be `@property (nonatomic) NSUInteger connectionCount;` and compiler will match it. there is no need to use `internalConnectionCount`.